jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-dry-run.md

```diff
@@ -0,0 +1,24 @@
+# Test a worker
+
+## Overview
+
+Dry run a worker. The worker needs to be initialized before running this command.
```

Comment:
add link to init. I think this statement is true for most of the commands, please add it to all command where it's needed